org/teams.yaml

```diff
@@ -57,6 +57,7 @@ teams:
       - SataQiu
       - sbezverk
       - sdake
+      - shankgan
```

Comment:
This list is for developers that are a maintainer of a working group, so I don't think you should add yourself here until you become a maintainer

Reply:
Which is hopefully soon :slightly_smiling_face:

Reply:
oops..thanks for the clarification!